aws_auditing_list.py

```diff
@@ -38,376 +44,678 @@ def default(self, o):
-                    self.dictionary[region_name]["ELB"][l['LoadBalancerName']]["instanceId"].append(id["InstanceId"])
-                
-                if not len(self.dictionary[region_name]["ELB"][l['LoadBalancerName']]["instanceId"]) > 0:
-                    unattached_instances = unattached_instances + 1
-                if len(self.dictionary[region_name]["ELB"][l['LoadBalancerName']]["instanceId"]) > 0:
-                    attached_instances = attached_instances + 1
-                
-                total_instances = attached_instances + unattached_instances
-
-                for term in elb_pricing[region_name]['ELB']['OnDemand']:
-                    elb_price = float(elb_pricing[region_name]['ELB']['OnDemand']['USD'])
-                    total_elb_cost = round(float( elb_price * total_instances*730.5), 3)
-                
-                elb_cost_region[region_name] = {
-                    "total_instances": total_instances,
-                    "price": elb_price,
-                    "total_elb_cost": total_elb_cost
-                }
-
-            self.cELB = elb_cost_region
+            total_instances = len(
+                lb["LoadBalancerDescriptions"]
+            )
+
+            for l in lb["LoadBalancerDescriptions"]:
+                self.dictionary[region_name]["ELB"][
+                    l["LoadBalancerName"]
+                ] = {"instanceId": []}
+
+                if l["Instances"]:
+                    self.dictionary[region_name]["ELB"][
+                        l["LoadBalancerName"]
+                    ]["instanceId"] = [
+                        id for id in l["Instances"]
+                    ]
+                else:
+                    self.dictionary[region_name]["ELB"][
+                        l["LoadBalancerName"]
+                    ]["instanceId"] = []
+
+            for term in elb_pricing[region_name]["ELB"][
+                "OnDemand"
+            ]:
+                elb_price = float(
+                    elb_pricing[region_name]["ELB"][
+                        "OnDemand"
+                    ]["USD"]
+                )
+                total_elb_cost = round(
+                    float(
+                        elb_price * total_instances * 730.5
+                    ),
+                    3,
+                )
+
+            self.classic_elb[region_name] = {
+                "total_instances": total_instances,
+                "price": elb_price,
+                "total_elb_cost": total_elb_cost,
+            }
 
-    def get_networkELB_resources(self, regions):
-        elbv2_cost_region = {}
-        with open("elbv2_pricing.json","r") as elb2:
+    def get_network_elb_resources(self, regions):
+        with open("elbv2_pricing.json", "r") as elb2:
             elbv2_pricing = json.load(elb2)
 
         for region_name in regions:
-            total_elbv2_cost = 0
-            elbv2_cost_region[region_name] = {}
+            TOTAL_ELBV2_COST = 0
```

Comment:
Generally capitalized variables are constant and don't change, should this be all lower case since it changes? I don't think you need to initialize this since you are assigning it with values below that should be 0 if anything.